### ISSUE TITLE
cloudflared won't launch on Raspberry Pi 3 running on Ubuntu 18.04

### ISSUE THREAD

**Issue by YamiYukiSenpai** [cloudflare/cloudflared#115]
I downloaded the ARM version of `cloudflared`, added the `arm` architecture, and tried launching it, but it wouldn't work.

```
$ cloudflared -v
-bash: /usr/local/bin/cloudflared: No such file or directory
```
I tried installing the .deb file and the tarball file from [the PiHole's documentation](https://docs.pi-hole.net/guides/dns-over-https/).

**Comment by bcatubig** [comment]
I was able to build this on WSL (or any linux distro with Go). I used `scp` to copy it to my server

## On WSL
```shell
# Get the source
go get -v github.com/cloudflare/cloudflared/cmd/cloudflared

# Build for arm64
GOOS=linux GOARCH=arm64 go build -v -x github.com/cloudflare/cloudflared/cmd/cloudflared

# Copy to Raspberry Pi
scp cloudflared <user>@<Raspberry Pi IP>:~/cloudflared
```

## On the Raspberry Pi

```shell

# Verify the binary works
./cloudflared -v
cloudflared version DEV (built unknown)

# Copy the bin to system executable dir
sudo mv cloudflared /usr/bin/

# Verify Again
$ cloudflared -v
cloudflared version DEV (built unknown)
```

hope this helps

**Comment by jnozsc** [comment]
it will be nicer if we could have an official arm64/armhf build, since ubuntu official support Raspberry Pi 

- https://ubuntu.com/download/raspberry-pi
- https://wiki.ubuntu.com/ARM/RaspberryPi

> Note that the information on this page currently only applies to the (ARMv7 and ARMv8) Raspberry Pis: 2B, 3B, 3A+, 3B+, 4B, Compute Module 3, and Compute Module 3+. The original (ARMv6) based Raspberry Pis, including the A, B, B+, 0 and 0W, are not supported under Ubuntu.

the build command works for me. thanks for the comment

**Comment by amingilani** [comment]
Can confirm that this issue still persists on alpine Linux today. But this time with the tar file. I had the following in my `Dockerfile`:

```
wget -O cloudflared.tgz https://bin.equinox.io/c/VdrWdbjqyF/cloudflared-stable-linux-arm.tgz \
 && tar -xzvf cloudflared.tgz \
 && rm cloudflared.tgz \
 && chmod +x cloudflared
```

Execution yields:

```
bash-5.0# ./cloudflared
bash: ./cloudflared: No such file or directory
```

But I can confirm that it exists in the same directory:

```
bash-5.0# ls | grep cloudflared
cloudflared
```

**Comment by sodabrew** [comment]
edc6969 was merged in time for the 2020.8.0 release, please see https://github.com/cloudflare/cloudflared/releases for recent releases with an arm64 build.
